internal/pkg/model/naming.go

```diff
@@ -305,38 +310,48 @@ func (n Naming) CodeFileExt(componentId string) string {
 	}
 }
 
-func (n Naming) VariablesPath(parentPath string, config *Config) PathInProject {
```

Comment:
Pred tym tu boli  metody `VariablesPath` a `VariablesValuesPath`.
Neuvedomil som si, ze `VariablesConfig` a `VariablesValuesConfigRow` su iba specialnym pripadom `config/row` .... a teda nemozu mat vlastnu metodu v `Naming`, lebo sa to tazko napaja na zbytok apky.